cypress/support/fragments/data-export/dataExportResults.js

```diff
@@ -13,5 +13,24 @@ export default {
       }),
       MultiColumnListCell({ 'row': 0, 'column': defaultJobProfile }).exists(),
     ]);
-  }
+  },
+
+  verifySuccessExportResultCells() {
+    const dateString = /\d{1,2}\/\d{1,2}\/\d{4},\s\d{2}:\d{2}\s\w{2}/gm;
+    return cy.do([
+      MultiColumnListCell({ 'row': 0, columnIndex: 1 }).is({ content: 'Completed' }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 2 }).perform(element => {
+        expect(element.innerText).to.match(/\d{1,4}/gm);
```

Comment:
"Total:  contains the number of UUIDs in the file that triggered the export" - add implementation of this expectation